.gitignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
You could've added the unused files on this file, So this could ignore files like "Social Media Icons" and "Header-pic.png" since you aren't using them.